sources/assets/exegol/load_supported_setups.sh

```diff
@@ -131,6 +131,33 @@ function deploy_firefox_addons() {
   fi
 }
 
+function deploy_bloodhound_customqueries_replacement() {
+  # Merge all the *.json files in customqueries format from my-resources/.../replacement to replace the Exegol provided one
+  if \
+    [[ -d "$MY_Setup_PATH/bloodhound_customqueries/replacement" ]] && \
+    [[ -n $(find "$MY_Setup_PATH/bloodhound_customqueries/replacement" -type f -name "*.json") ]]; then
+      bqm --verbose --ignore-default --output-path /root/.config/bloodhound/customqueries.json.bqm -i "$MY_Setup_PATH/bloodhound_customqueries/replacement"
+      [[ -f "/root/.config/bloodhound/customqueries.json.bqm" ]] && mv "/root/.config/bloodhound/customqueries.json.bqm" "/root/.config/bloodhound/customqueries.json"
+  fi
+}
+
+function deploy_bloodhound_customqueries_merge() {
+  # Merge all the *.json files in customqueries format from my-resources/.../merge with the Exegol provided one
+  if \
+    [[ -d "$MY_Setup_PATH/bloodhound_customqueries/merge" ]] && \
+    [[ -n $(find "$MY_Setup_PATH/bloodhound_customqueries/merge" -type f -name "*.json") ]]; then
+      bqm --verbose --ignore-default --output-path /root/.config/bloodhound/customqueries.json.bqm -i "$MY_Setup_PATH/bloodhound_customqueries/merge,/root/.config/bloodhound/customqueries.json"
+      [[ -f "/root/.config/bloodhound/customqueries.json.bqm" ]] && mv "/root/.config/bloodhound/customqueries.json.bqm" "/root/.config/bloodhound/customqueries.json"
+  fi
+}
+
+function deploy_bloodhound_customqueries() {
+  # If a user places Bloodhound json files in both folders my-resources/.../merge and my-resources/.../replacement,
+  # replacement must be executed last to only keep the output of replacement.
```

Comment:
Is there a bloodhound config file ? If so, we can also offer the option to the user to overwrite the config file.

Reply:
Agreed, I added support for it as well in the latest commit.